pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2576,7 +2626,6 @@
     ::  assert temporal coherence
     ::
     ?<  =(~ next-wake.state)
-    ?>  (gte now.channel (need next-wake.state))
```

Comment:
That was the idea, although since I made some changes, I wonder if I could reinstate this assertion -- +on-prod should actually lawfully reset the .next-wake now, instead of the original behavior, which updated state but then threw it away.  Unsurprisingly, that was easy to get wrong, so I added a new codepath in the packet-pump for +on-prod that doesn't rely on hacking +on-wake the way the original one did.

I'll try reinstating the assertion and seeing if it still works.